### PR TITLE
Add utility for postgresql + noobaa deployment with external pgsql

### PR DESCRIPTION
**Pull request by petr-balogh**
Ticket:
https://issues.redhat.com/browse/OCSQE-2046

### CODE REVIEW

**Comment by petr-balogh** [comment]
TODO: Do remove database in destroy

File: conf/ocsci/noobaa_external_pgsql.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+ENV_DATA:
+  noobaa_external_pgsql: True
+# Please make sure you create custom config file or update data/auth.yaml
+# file with following configuration values:
+#
+# AUTH:
+#   host: my.pgsql.server
+#   username: pgsql_user  # Needs to have permission for create/delete DB.
+#   password: password_of_user
+#   port: 5432
```
[the remark this answers is not in the document]
Comment (petr-balogh) [reply conf/ocsci/noobaa_external_pgsql.yaml]: thanks for noticing. Done!

**Comment by petr-balogh** [comment]
> TODO: Do remove database in destroy

Done in last pushed code

**Review by dahorak — APPROVED** [review]

**Review by vavuthu — APPROVED** [review]

**Comment by petr-balogh** [comment]
There is one more opened issue:
https://github.com/red-hat-storage/ocs-ci/issues/9487

Which will need some helper utils/function/tests adjustment as some code refer to noobaa db pod - which in this case of deployment doesn't exist.

**Pull request merged**